tests/test_xsd.py

```diff
@@ -92,46 +108,58 @@ def test_xsd():
     """)
 
     xml_doc = run_and_parse(testdir).xml_doc
-    xmlschema = etree.XMLSchema(etree.parse(get_xsd('mk8s')))
+    xmlschema = etree.XMLSchema(etree.parse(xmlpf._get_xsd('mk8s')))
 
     # Test
     xmlschema.assertValid(xml_doc)
 
 
-def test_missing_required_marks_asc(testdir, undecorated_test_function):
+def test_missing_required_marks_asc(testdir, undecorated_test_function, mocker):
     """Verify that XSD will enforce the presence of 'test_id' and 'jira_id' properties for test cases for the 'asc'
     CI environment.
     """
 
+    # Mock
+    zz = mocker.MagicMock()
+    xmlpf = XmlParsingFacade(zz)
+
     # Setup
     testdir.makepyfile(undecorated_test_function.format(test_name='test_typo_global'))
 
     xml_doc = run_and_parse(testdir).xml_doc
-    xmlschema = etree.XMLSchema(etree.parse(get_xsd('asc')))
+    xmlschema = etree.XMLSchema(etree.parse(xmlpf._get_xsd('asc')))
 
     # Test
     assert xmlschema.validate(xml_doc) is False
 
 
-def test_missing_required_marks_mk8s(testdir, undecorated_test_function):
+def test_missing_required_marks_mk8s(testdir, undecorated_test_function, mocker):
     """Verify that XSD will enforce the presence of 'test_id' and 'jira_id' properties for test cases  for the 'mk8s'
     CI environment.
     """
 
+    # Mock
+    zz = mocker.MagicMock()
+    xmlpf = XmlParsingFacade(zz)
+
     # Setup
     testdir.makepyfile(undecorated_test_function.format(test_name='test_typo_global'))
 
     xml_doc = run_and_parse(testdir).xml_doc
-    xmlschema = etree.XMLSchema(etree.parse(get_xsd('mk8s')))
+    xmlschema = etree.XMLSchema(etree.parse(xmlpf._get_xsd('mk8s')))
```

Comment:
Since an external program is accessing this method it should be marked as public.